### PR TITLE
Website

### PR DESCRIPTION
**Pull request by Aurelsicoko**
Add the website associated with the library. The website is responsive and follows the design provided by the design team.

The website and storybook are served by a dead-simple Koa application.

### CODE REVIEW

File: website/src/components/footer.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    style={{
+      height: 160,
+      padding: '36px 0',
+      backgroundColor: '#ffffff'
+    }}
+  >
+    <div className="container-fluid">
+      <div className="row">
+        <div className="col-12" style={{ textAlign: 'center' }}>
+        <a
```
Comment (virginieky) [inline website/src/components/footer.js]: indent

File: website/src/components/footer.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+              fontFamily: 'Lato'
+          }}>
+          Made with love by <a href="https://www.strapi.io" target='_blank' rel='noopener noreferrer' style={{ textDecoration: 'none', color: '#0B269C' }}>Strapi</a></p>
+        </div>
+      </div>
+    </div>
+  </footer>
+);
+
+Footer.propTypes = {
```
Comment (virginieky) [inline website/src/components/footer.js]: Same, no props used in Footer

File: website/src/components/h3.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        width: 36,
+        height: 6,
+        backgroundColor: '#0B269C'
+      }}
+    ></span>
+    {children}
+  </h3>
+)
+
+H3.propTypes = {
```
Comment (virginieky) [inline website/src/components/h3.js]: same

File: website/src/components/h4.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      marginBottom: 15,
+      ...style
+    }}
+  >
+    {children}
+  </h4>
+)
+
+H4.propTypes = {
+  siteTitle: PropTypes.string,
```
Comment (virginieky) [inline website/src/components/h4.js]: same

File: website/src/components/navigation.js [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </nav>
+)
+
+Navigation.propTypes = {
+  siteTitle: PropTypes.string,
```
Comment (virginieky) [inline website/src/components/navigation.js]: props not used

File: website/src/components/paragraph.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+      textAlign: 'left',
+      ...style
+    }}
+  >
+    {children}
+  </p>
+)
+
+Paragraph.ParagraphpropTypes = {
+  siteTitle: PropTypes.string,
```
Comment (virginieky) [inline website/src/components/paragraph.js]: same

File: website/src/components/seo.js [hunk 7]
```diff
@@ -77,7 +77,18 @@ function SEO({ description, lang, meta, keywords, title }) {
...
+      <link
+        rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css"
+      />
+      <link
+        rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.3.1/css/bootstrap-grid.min.css"
+      />
+      {children}
+      {/* <link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Lato:400,700,900' /> */}
```
Comment (virginieky) [inline website/src/components/seo.js]: remove comment ?

File: website/gatsby-config.js [hunk 8]
```diff
@@ -25,7 +27,18 @@ module.exports = {
         background_color: `#663399`,
         theme_color: `#663399`,
         display: `minimal-ui`,
-        icon: `src/images/gatsby-icon.png`, // This path is relative to the root of the site.
+        icon: `src/images/favicon.png`, // This path is relative to the root of the site.
```
[the remark this answers is not in the document]
Comment (Aurelsicoko) [reply website/gatsby-config.js]: The comment seems relevant, it gives true information, it's better to have a comment than nothing 🤷‍♂️

**Comment by Aurelsicoko** [comment]
I will do the feedback but we should not talk in the PR review about the indentation. We have tools for that, I didn't take the time to setup Prettier or a proper linter but it's just to reminder to say that's not part of our job 👍 

Btw, I'm going to add a linter to avoid this situation in the future :)

File: website/package.json [hunk 9]
```diff
@@ -1,27 +1,43 @@
 {
-  "name": "gatsby-starter-default",
+  "name": "buffet-website",
   "private": true,
-  "description": "A simple starter to get up and developing quickly with Gatsby",
+  "description": "The website to introduce the Buffet.js library and the concepts/philosophy behind it",
```
Comment (soupette) [inline website/package.json]: website which presents the Buffet.js library and its concepts/philosophy

**Pull request merged**